tests/Microsoft.Identity.Web.Test/Certificates/DefaultCertificateLoaderTests.cs

```diff
@@ -16,7 +16,9 @@ public class DefaultCertificateLoaderTests
         // [InlineData(CertificateSource.Path, @"c:\temp\WebAppCallingWebApiCert.pfx", "")]
         // [InlineData(CertificateSource.StoreWithDistinguishedName, "CurrentUser/My", "CN=WebAppCallingWebApiCert")]
         // [InlineData(CertificateSource.StoreWithThumbprint, "CurrentUser/My", "962D129A859174EE8B5596985BD18EFEB6961684")]
+#pragma warning disable xUnit1012 // Null should only be used for nullable parameters
```

Comment:
Same q here and on the subsequent xUnit1012 disables

Reply:
same reply as [here](https://github.com/AzureAD/microsoft-identity-web/pull/3053/files#r1785076499)